docker/servicesmw/site.conf

```diff
@@ -0,0 +1,29 @@
+server {
+    server_name mediawiki.dev.sjc.k8s.wikia.net;
```

Comment:
This file is not used. If it were used I think I would not work. Ingress listens on address `${SERVICESMW_NAME}.wikia.com` and nginx expects `mediawiki.dev.sjc.k8s.wikia.net`